Products/ZenHub/zenhubworker.py

```diff
@@ -54,8 +54,8 @@ class zenhubworker(ZCmdBase, pb.Referenceable):
     "Execute ZenHub requests in separate process"
 
     def __init__(self):
+        signal.signal(signal.SIGUSR2, signal.SIG_IGN)
```

Comment:
The subsequent call to register the original SIGUSR2 handler is still there. SIGUSR2 is only ignored until the __init__() methods of all it's super classes along with the pre-requisite setup for zenhubworker.sighandler_USR2 is run.